src/Uno.Templates/content/unoapp/azure-pipelines.yml

```diff
@@ -0,0 +1,75 @@
+###################################################
+#                    IMPORTANT                    #
+# Please make sure  GitTools is installed in your #
+# project. Otherwise all jobs will fail           #
+# See how to install new extensions here:         #
+# https://learn.microsoft.com/en-us/azure/devops  #
```

Comment:
Let's remove this warning and anything else that's not used in the current pipeline